### PR TITLE
Simplification in tests and fixing docu

### PR DESCRIPTION
**Pull request by kobliha**

### CODE REVIEW

File: gloves-firewall/examples/susefirewall2_conf [hunk 1]
```diff
@@ -28,7 +28,11 @@ begin
     puts susefirewall2.inspect
 
     if susefirewall2.nil?
+<<<<<<< HEAD:yast++lib-firewall/examples/susefirewall2_conf
+      error = YLib::Susefirewall2::last_error
+=======
       error = Glove::Susefirewall2::last_error
+>>>>>>> upstream/master:gloves-firewall/examples/susefirewall2_conf
```
Comment (jreidinger) [inline gloves-firewall/examples/susefirewall2_conf]: this is wrong. You have there unmerged changes

**Comment by jreidinger** [comment]
nice patch. Just Please solve conflict

File: gloves-firewall/examples/susefirewall2_conf (hunk 1 above)
Comment (mchf) [inline gloves-firewall/examples/susefirewall2_conf]: this seems strange to me ;-)

Comment (kobliha) [inline gloves-firewall/examples/susefirewall2_conf]: Yep, probably something left from April or so

**Comment by kobliha** [comment]
Conflict has been fixed

**Pull request merged**